src/main/java/com/minecolonies/structures/client/TemplateBlockAccessTransformHandler.java

```diff
@@ -0,0 +1,40 @@
+package com.minecolonies.structures.client;
+
+import net.minecraft.world.gen.structure.template.Template;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public class TemplateBlockAccessTransformHandler
+{
+    private static TemplateBlockAccessTransformHandler ourInstance = new TemplateBlockAccessTransformHandler();
+
+    public static TemplateBlockAccessTransformHandler getInstance()
+    {
+        return ourInstance;
+    }
+
+    private Map<Predicate<Template.BlockInfo>, Function<Template.BlockInfo, Template.BlockInfo>> blockInfoTransformHandler = new HashMap<>();
+
+    private TemplateBlockAccessTransformHandler()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1213)